include/deal.II/base/geometry_info.h

```diff
@@ -196,9 +214,17 @@ struct RefinementPossibilities<1>
    */
   enum Possibilities
   {
-    no_refinement= 0,
+    /**
+     * Do not refine.
+     */
+    no_refinement = 0,
+    /**
+     * Perform a cut in the x-direction.
+     */
     cut_x        = 1,
```

Comment:
The patch is already merged, but if you felt ambitious: I always thought that this is a poorly chosen name because it's not clear to me whether the "cut is parallel to the x-axis" or whether the cell us subdivided into a part that lies further west and a part that lies further east on the x-axis". The latter is the case. It may be nice to reflect this in the documentation, but I'm not feeling strongly that this needs to be fixed now.